.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             apt-get -y install curl
             pip install --upgrade pip
             pip install poetry
-            poetry install
+            poetry install --no-ansi
```

Comment:
Seems to be a bug in the latest `poetry==1.3` that was causing this install command to error out. This issue thread describes the situation, and the `--no-ansi` was a solution identified in the thread: https://github.com/python-poetry/poetry/issues/7184